.gitignore

```diff
@@ -1,5 +1,7 @@
 # ## Linux
-.*
+.cache
+.coverage
+build/
 !.gitignore
```

Comment:
remove this as well, now without '.*' it isn't useful

Reply:
Fixed